bridge/irc_manager.go

```diff
@@ -357,6 +357,11 @@ func (m *IRCManager) SendMessage(channel string, msg *DiscordMessage) {
 			IsAction:   msg.IsAction,
 		}
 
+		if strings.HasPrefix(line, "/me") {
```

Comment:
The prefix check should be for `"/me "` instead.

And it should make sure that the length is greater than 4, otherwise someone could send `/me ` on its own to send an empty message (which wouldn't go through)

Reply:
Good catch, "/meblahblah" definitely isn't the same as "/me blah blah" and don't want to convert the former. I'll fix this to check the length and use the proper detection.

Reply:
Shouold be fixed in the latest commit.